Util/Tokenization/src/main/java/org/tribuo/util/tokens/impl/wordpiece/WordpieceTokenizer.java

```diff
@@ -194,12 +225,11 @@ public TokenType getType() {
     // TODO!
     @Override
     public WordpieceTokenizer clone() {
-        throw new RuntimeException(new CloneNotSupportedException());
-//      try {
-//          WordpieceTokenizer copy = (WordpieceTokenizer) super.clone();
-//          return copy;
-//      } catch (CloneNotSupportedException e) {
-//          throw new AssertionError("WordpieceTokenizer is Cloneable, but clone call failed");
-//      }
+      try {
+          WordpieceTokenizer copy = (WordpieceTokenizer) super.clone();
+          return copy;
```

Comment:
This should ensure the tokenizer is ready to receive new input and not connected to the old token stream.

Reply:
ok - I don't have a strong intuition about this.  But I reset all the state-related variables of copy.